client/src/components/generic/WorkInProgressComponent/WorkInProgressComponent.tsx

```diff
@@ -0,0 +1,18 @@
+import Lock from "@/assets/icons/lock.svg"
+import Link from "next/link"
+const WorkInProgressComponent = () => {
+  return (
+    <div className="flex w-full flex-col items-center justify-center gap-4">
+      <Lock className="fill-dark-blue-100" />
+      <h2 className="italic">Events Page Unavailable</h2>
```

Comment:
Is this still generic if the name is hardcoded?

Reply:
yep, good point. I thought we only need this for events